scripts/setup-docker.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+export BROKER_URL=amqp://olympia:olympia@rabbitmq/olympia
+export DATABASE_URL=mysql://root:@mysqld/olympia
+export ELASTICSEARCH_LOCATION=elasticsearch:9200
+export MEMCACHE_LOCATION=memcached:11211
+export MYSQL_DATABASE=olympia
+export MYSQL_ROOT_PASSWORD=docker
+export OLYMPIA_SITE_URL
+export PYTHONDONTWRITEBYTECODE=1
+export PYTHONUNBUFFERED=1
+export RECURSION_LIMIT=10000
+export REDIS_LOCATION=redis://redis:6379/0?socket_timeout=0.5
+export TERM=xterm-256color
+
+# initialize_db:
```

Comment:
I can't call `initialize_db` directly because `reset_db` and `createsuperuser` include prompts.